### PR TITLE
Reverse colors in scale_*_distiller and add direction support for brewer/distiller

### LINKED ISSUE #357: reverse brewer color scale?
In R you can reverse the palette min-max:
scale_fill_brewer(palette = "Greys",
                                            direction = -1)

what alternative can I have in plotnine for a brewer palette?

### LINKED ISSUE #377: scale_*_distiller diverging palette colors are reversed by default
Hi,

This code 

```py
ggplot(aes(x='gene', y='tissue'), data=df) +
    geom_point(aes(size='neglog_pval_adj', fill='logodds', color='significant'))+
    scale_fill_distiller(type='div', limits=limit, name='log odds')
```

produces this fill legend:

image

However, as we see in colorbrewer the diverging palette looks like the reverse (i.e. brown first, green last)

image

Am I wrong?

When I run:

```py
import mizani

mizani.palettes.brewer_pal(type='div')(7)
```

I get:

```
['#8C510A', '#D8B365', '#F6E8C3', '#F5F5F5', '#C7EAE5', '#5AB4AC', '#01665E']
```

which looks correct, so I'm a bit puzzled why colors are flipped in my plot. Is this related to the direction argument of `mizani.palettes.brewer_pal`?

(I'm using mizani 0.6.0 and plotnine 0.6.0)

### PR DESCRIPTION
**Pull request by gokceneraslan**
Fixes #377. Fixes #357 .

### CODE REVIEW

File: plotnine/scales/scale_color.py [hunk 1]
```diff
@@ -357,7 +357,10 @@ def __init__(self, type='seq', palette=1, values=None, **kwargs):
         # Grab 7 colors from brewer and create a gradient palette
         # An odd number matches the midpoint of the palette to that
         # of the data
-        colors = brewer_pal(type, palette)(7)
+        # See https://github.com/has2k1/plotnine/issues/377
+        # TODO: make direction parameter -1 by default: 
+        # https://github.com/tidyverse/ggplot2/blob/master/R/scale-brewer.r#L89
+        colors = reversed(brewer_pal(type, palette)(7))
```
Comment (has2k1) [inline plotnine/scales/scale_color.py]: You can add the `direction` parameter to scale colour brewer. Support for it was added in [mizani v0.6.0](https://mizani.readthedocs.io/en/stable/palettes.html#mizani.palettes.brewer_pal), I just forgot to add it to plotnine right after releasing mizani.

Comment (gokceneraslan) [reply plotnine/scales/scale_color.py]: Done. How do you do the dependencies? Is it ok to add mizani=0.6.0 dependency?

Comment (gokceneraslan) [reply plotnine/scales/scale_color.py]: Oh mizani >= 0.6.0 is already a requirement, sweet.

**Comment by gokceneraslan** [comment]
@has2k1 looks good?

**Comment by has2k1** [comment]
@gokceneraslan, thank you.

**Pull request merged**